antenna/app.py

```diff
@@ -292,8 +291,6 @@ def get_app(config=None):
     if config is None:
         config = ConfigManager(
             environments=[
-                # Pull configuration from env file specified as ANTENNA_ENV
-                ConfigEnvFileEnv([os.environ.get('ANTENNA_ENV')]),
                 # Pull configuration from environment variables
                 ConfigOSEnv()
```

Comment:
I could probably simplify this further and just do `ConfigManager.basic_config()`. That sets up a `ConfigManager` with `ConfigEnvFileEnv('.env')` and `ConfigOSEnv()`. We pull in `my.env` via docker-compose and I didn't think I wanted to add the `.env` thing to the server environments--there should be only one way to configure things.

Anyhow, so I didn't do that and set it up explicitly like this.